### ISSUE TITLE
Both GUI and CLI versions are not working on any versions of Windows. 

### ISSUE THREAD

**Issue by swerfot** [ma1co/Sony-PMCA-RE#84]
Spent countless hours trying to hunt down all the required dependencies and their versions, finally gave up. Every time I solve one error by installing some Python dependency, another, new one pops up. Not worth the time. Tried to make it work on Windows 10, Windows 7 and Windows XP. Failed on all of these.

**Comment by ma1co** [comment]
Running `pip -r requirements.txt` should be enough. Otherwise there are Windows [binaries](https://github.com/ma1co/Sony-PMCA-RE/releases/latest) provided. If you have any suggestions or well-defined problems, feel free to re-open the issue.

**Comment by swerfot** [comment]
Yep, the problem is, I was trying to use exactly those Windows binaries that you provided. And they would not work, because based on error messages I had to go hunting for a gazillion dependencies. First install Python, then go download loads of additional components for Python to resolve every new error message... After long and tiresome troubleshooting I finally managed to run those binaries under a freshly installed 32-bit Windows 7 virtual machine + Python + MS Visual C++ runtime. That was the only combination that worked: specifically 32 bit Windows 7 with those runtimes. Your Windows binaries will not work on any other version of Windows, and even 64-bit Windows 7 is a problem. The only OS it works under is Windows 7 32-bit.
